src/js/layout.js

```diff
@@ -15,9 +15,11 @@ const processClassName = (data, field) => {
     }
 
     // Now that the col- types were lifted, remove from the actual input field
-    for (let index = 0; index < classes.length; index++) {
-      const element = classes[index]
-      field.classList.remove(element)
+    if (field.classList) {
+      for (let index = 0; index < classes.length; index++) {
+        const element = classes[index]
+        field.classList.remove(element)
+      }
```

Comment:
this is quite old and could probably be refactored to:
```javascript
field.classList.remove(...field.classList)
```

Reply:
`field.classList.remove(...classes)` seemed to work well